### ISSUE TITLE
Don't install 2.0.0.1

### ISSUE THREAD

**Issue by free6k** [colbylwilliams/Xamarin.TTTAttributedLabel#6]
I have next problem

```
Could not install package 'Xamarin.TTTAttributedLabel 2.0.0.1'. You are trying to install this package into a project that targets 'Xamarin.iOS,Version=v1.0', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.
```

Version 2.0.0.0 installed without problem.

**Comment by AliakseiT** [comment]
@colbylwilliams, the slashes in your nuget build script are likely broken.

**Comment by franzdevel** [comment]
Hi, I am having the same problem when trying to install 2.0.0.1. I can install 2.0.0 but I then I would probably run into the bitcode issue (?). Is there any chance to get the problem with 2.0.0.1 fixed any time soon?

**Comment by Emasoft** [comment]
Same problem here:

> Could not install package 'Xamarin.TTTAttributedLabel 2.0.0.1'. You are trying to install this package into a project that targets 'Xamarin.iOS,Version=v1.0', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.

**Comment by colbylwilliams** [comment]
This is fixed in [version 2.0.0.2](https://www.nuget.org/packages/Xamarin.TTTAttributedLabel/2.0.0.2).
